roles/download/defaults/main.yml

```diff
@@ -81,7 +81,7 @@ cni_version: "v0.8.1"
 weave_version: 2.5.2
 pod_infra_version: 3.1
 contiv_version: 1.2.1
-cilium_version: "v1.5.5"
+cilium_version: "v1.6.4"
```

Comment:
v1.6.5 has been released in the meantime. Could you please bump the version here?

Reply:
Done!

Reply:
Mmh, now 1.6.6 is out already :roll_eyes: